include/swift/AST/Expr.h

```diff
@@ -3355,12 +3355,20 @@ class SequenceExpr final : public Expr,
 
   SequenceExpr(ArrayRef<Expr*> elements)
     : Expr(ExprKind::Sequence, /*Implicit=*/false) {
+    // Make sure all elements in the SequenceExpr an not null
+    for (auto element : elements) {
+      assert(element);
```

Comment:
Quick question - in no-assertions builds - is this going to result in the warning that loop is empty? It seems like it might be best to do something like `assert (llvm::find_if(elements, [](const Expr *expr) { return !expr; }) == elements.end());` instead, WDYT?

Reply:
I was hoping the optimiser would remove the loop since I couldn't think of a way to do it in one statement. I'll switch to your implementation. That's definitely better. Thanks!

Reply:
Yes, great suggestion by @xedin'; avoid depending on "the optimizer will hopefully remove this" for assertion-specific code.